src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -1486,6 +1486,13 @@ namespace {
                 auto gas_target = record.getItem("GAS_TARGET").get<UDAValue>(0);
                 auto water_target = record.getItem("WATER_TARGET").get<UDAValue>(0);
                 auto liquid_target = record.getItem("LIQUID_TARGET").get<UDAValue>(0);
+                double guide_rate = -1;
```

Comment:
The whole guide rate machinery should be ignored if the first item - groupname - equals the special string `"FIELD"`.